systems/money.go

```diff
@@ -0,0 +1,116 @@
+package systems
+
+import (
+	"fmt"
+	"image/color"
+	"sync/atomic"
+
+	"engo.io/ecs"
+	"engo.io/engo"
+	"engo.io/engo/common"
+	"github.com/EngoEngine/TrafficDefense/systems/ui"
```

Comment:
Why is this TrafficDefense and not TrafficManager? Also should we add an engo.io/*** import for traffic manager?